src/commands/force/data/soql/query.ts

```diff
@@ -35,45 +35,54 @@ const commonMessages = Messages.loadMessages('@salesforce/plugin-data', 'message
  * Will collect all records and the column metadata of the query
  */
 export class SoqlQuery {
-  public async runSoqlQuery(connection: Connection, query: string, logger: Logger): Promise<SoqlQueryResult> {
-    const config: SfdxConfigAggregator = await SfdxConfigAggregator.create();
-
+  public async runSoqlQuery(
+    connection: Connection,
+    query: string,
+    logger: Logger,
+    configAgg: SfdxConfigAggregator
+  ): Promise<SoqlQueryResult> {
     let columns: Field[] = [];
     logger.debug('running query');
 
-    // take the limit from the config, then default 10,000
+    // take the limit from the config, then default 50,000
     const queryOpts: Partial<QueryOptions> = {
       autoFetch: true,
-      maxFetch: (config.getInfo('maxQueryLimit').value as number) || 10000,
+      maxFetch: (configAgg.getInfo('maxQueryLimit').value as number) || 50000,
```

Comment:
the default limit in sfdx-core was 10000 but plugin-data always set it to 50000, changed to 10000 here: https://github.com/salesforcecli/plugin-data/pull/306/files#diff-1a06fd5095e4d783ae9b6e36afb369a23a237384d219007547e157467dd7f4e7L39